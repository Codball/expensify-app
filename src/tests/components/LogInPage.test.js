import React from 'react';
import { shallow } from 'enzyme';
import { LogInPage } from '../../components/LogInPage';

test('should correctly render LogInPage', () => {
    const wrapper = shallow(<LogInPage />);
    expect(wrapper).toMatchSnapshot();
});

test('should call startLogin on button click', () => {
    const startLogin = jest.fn();
    const wrapper = shallow(<LogInPage startLogin={startLogin} />);
    wrapper.find('button').simulate('click');
    expect(startLogin).toHaveBeenCalled();
});
