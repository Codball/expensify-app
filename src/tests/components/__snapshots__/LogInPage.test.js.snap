// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render LogInPage 1`] = `
<div>
  <button>
    Login
  </button>
</div>
`;
